### PR TITLE
[modbus] reduce log level when modbus slave returns DEVICE_BUSY exception

### PR DESCRIPTION
**Pull request by nagisa**
This exception is meant to indicate that the request should be retried shortly, essentially, and at least some of the devices I own seem to be busy bees. Thus my logs receive significant spam of this warning.

Since the exception is transient and retrying it is the expected course of action, I think it makes sense to reduce the log level here slightly and only output an error when the retries get exhausted.

### CODE REVIEW

File: bundles/org.openhab.core.io.transport.modbus/src/main/java/org/openhab/core/io/transport/modbus/internal/ModbusManagerImpl.java [hunk 1]
```diff
@@ -662,7 +668,11 @@ private <R, C extends ModbusResultCallback, F extends ModbusFailureCallback<R>,
                 } catch (ModbusSlaveException e) {
                     lastError.set(new ModbusSlaveErrorResponseExceptionImpl(e));
                     // Slave returned explicit error response, no reason to re-establish new connection
-                    if (willRetry) {
+                    if (willRetry && e.getType() == MODBUS_EXCEPTION_SLAVE_DEVICE_BUSY) {
+                        logger.info(
```
Comment (J-N-K) [inline bundles/org.openhab.core.io.transport.modbus/src/main/java/org/openhab/core/io/transport/modbus/internal/ModbusManagerImpl.java:672]: When re-trying is an expected operation, you can reduce that to DEBUG.

**Review by J-N-K — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
@nagisa : did you see the review comment?

File: bundles/org.openhab.core.io.transport.modbus/src/main/java/org/openhab/core/io/transport/modbus/internal/ModbusManagerImpl.java (hunk 1 above)
Comment (nagisa) [inline bundles/org.openhab.core.io.transport.modbus/src/main/java/org/openhab/core/io/transport/modbus/internal/ModbusManagerImpl.java:672]:
```suggestion
                        logger.debug(
```

**Comment by lolodomo** [comment]
@J-N-K : you can probably merge this PR now.

**Review by J-N-K — APPROVED** [review]
Thanks! And sorry for letting you wait so long. I didn't see my comments were addressed.

**Pull request merged**